Northstar.Client/mod/scripts/vscripts/_custom_codecallbacks_client.gnut

```diff
@@ -87,16 +98,19 @@ void function CHudChat_OnReceivedSayTextMessageCallback(int fromPlayerIndex, str
         fromPlayerName = fromPlayer.GetPlayerNameWithClanTag()
     }
 
+    // Null player + isTeam == true: Server with no tag.
```

Comment:
I'm a bit confused as to why this is the case? I wouldn't say that I disagree with it, just curious

Reply:
There are no occasions where the server sends a team message. Using this combination is an easy way to mark a message as having no `[SERVER]` tag without having to break any compatibility with older versions, or changing the data that's sent.

Reply:
Bit of a shame to disallow servers to send a message to just a team (although they could just send a whisper server message to each player on that team rather simply) but yeah that makes sense.

Some docs on this would be good though, as it's maybe not the most intuitive thing in the world

Reply:
Yeah, once it's merged I'll make some docs. Also, right now, there's no way to even send a message to a team (unless you loop through players). So it really doesn't change anything.

Reply:
(you can make them already and just mark the docs PR as draft :P)

Reply:
Will do as soon as I have time.